.github/actions/store-delete-k8s-ghcr/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Delete GHCR Token
+description: Deletes the K8s secret used for pulling images from GHCR.
```

Comment:
These seem stale, "Store Docker credentials" or similar? This isn't GHCR specific.